source/utils/DistributionSampler.h

```diff
@@ -0,0 +1,50 @@
+// ----------------------------------------------------------------------------
+// nexus | DistributionSampler.h
+//
+// This class provides the utilities to randomly sample from an input
+// distribution
+//
+// The NEXT Collaboration
+// ----------------------------------------------------------------------------
+
+#ifndef DISTRIBUTION_SAMPLER
+#define DISTRIBUTION_SAMPLER
+
+#include <Randomize.hh>
+#include <G4Types.hh>
+
+namespace nexus {
+
+  /// Generator of random points in a sphere
+  class DistributionSampler
```

Comment:
Do we need to have a class for this? Maybe we could just use the functions with a specific namespace (such as `sampler`, or `distrsampler`). Same for `FileReaderCVS`.

Reply:
In fact, I'm thinking that `GetRandBinIndex()` and `Sample()` may well belong to the `RandomUtils` collection of functions, what do you think? The `CheckOutOfBound` functions are very general and they may be used also in a different context than distribution sampling, so they may belong to a list of general functions.